ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/util/ByteWrappedHasher.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.spotify.ratatool.samplers.util
+
+import java.nio.charset.Charset
+
+import com.google.common.hash.{Funnel, HashCode, Hasher}
+import com.google.common.io.BaseEncoding
+
+object ByteWrappedHasher {
```

Comment:
Nitpick, can we just call this ByteHasher?

Reply:
Done!